saleor/account/migrations/0086_auto_20240722_1753.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.25 on 2024-07-22 17:53
+
+import django.contrib.postgres.indexes
+import django.db.models.functions.text
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("account", "0085_address_validation_skipped"),
+    ]
+
+    operations = [
+        migrations.AddIndex(
+            model_name="user",
+            index=django.contrib.postgres.indexes.GinIndex(
+                django.contrib.postgres.indexes.OpClass(
+                    django.db.models.functions.text.Upper("first_name"),
```

Comment:
trgm indexes are case insensitive, no reason for this step.
Tip: check locally `SELECT show_trgm('Name')` vs `SELECT show_trgm(UPPPER('Name'))`